src/Sinch/Core/StreamExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+
+namespace Sinch.Core
+{
+    internal static class StreamExtensions
+    {
+        public static string ConvertToBase64(this Stream stream)
```

Comment:
Where are you using this method? I can't find any occurence in the diff displayed by github